Scripts/remove-frameworks-extra-archs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Loop through all the embeded frameworks.
+echo "Architecture(s) built for this product: $ARCHS."
+for f in $BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH/*.framework/ ; do
+
+    # Not all the frameworks correctly specify the CFBundleExecutable from their info.plist so default to naming convention.
```

Comment:
Some of Firebase framewoks don't have this so I added it to build it. I wonder if it breaks something.